Python/fileutils.c

```diff
@@ -2390,13 +2390,18 @@ _Py_find_basename(const wchar_t *filename)
    to be the end of the path. 'normsize' will be set to contain the
    length of the resulting normalized path. */
 wchar_t *
-_Py_normpath_and_size(wchar_t *path, Py_ssize_t size, Py_ssize_t *normsize)
+_Py_normpath_and_size(wchar_t *path, Py_ssize_t size, Py_ssize_t start, Py_ssize_t *normsize)
```

Comment:
Could you also document the start parameter in the docstring above?

Reply:
Yeah:

> 'start' is the position where to start normalizing.